Dockerfile

```diff
@@ -1,32 +1,46 @@
-FROM registry.access.redhat.com/ubi9/python-311@sha256:39bd10c0d80a127280095ff352f8a6881e61c39c2edbbdf5206e6508adc3b8e7 AS prod
-COPY --from=ghcr.io/astral-sh/uv:0.4.26@sha256:7775c60dca9cc5827c36757c32c75985244d8f31447565fa8147e2b2e11ad280 /uv /bin/uv
+#
+# Builder image
+# Keep version in sync with prod image below
+#
+FROM registry.access.redhat.com/ubi9/python-311@sha256:129391e5b291e29a3b24a3fdb281a56d7cb8297bdf8ba92bb5c8581b53084324 AS builder
```

Comment:
if we want to remove duplicate image tag in the same file, can extract another stage

```dockerfile
FROM registry.access.redhat.com/ubi9/python-311@sha256:129391e5b291e29a3b24a3fdb281a56d7cb8297bdf8ba92bb5c8581b53084324 AS base
# er-outputs-secrets version. keep in sync with pyproject.toml
LABEL konflux.additional-tags="0.2.1"
COPY LICENSE /licenses/MIT # do we need this in the final image?

FROM base AS builder
...

FROM builder AS test
...

FROM base AS prod
...
```

Reply:
Sure 🤦 Changed in 4a9222a

Reply:
BTW, the enterprise contract konflux check requires a license file in the image.

Reply:
currently only builder image contain license file, is it needed for prod one to pass konflux check on build master?

Reply:
yes, it's need for the prod image only 🤦